server/athenian/api/controllers/jira_controller/common.py

```diff
@@ -79,20 +75,16 @@ async def collect_account_info(
         op="sdb/ids",
     )
     repos = [str(r) for r in repos]
-    if with_branches_and_settings:
-        settings = Settings.from_request(request, account, prefixer)
-        (branches, default_branches), logical_settings = await gather(
-            BranchMiner.load_branches(
-                repos, prefixer, account, meta_ids, mdb, pdb, cache, strip=True,
-            ),
-            settings.list_logical_repositories(repos),
-            op="sdb/branches and releases",
-        )
-        repos = logical_settings.append_logical_prs(repos)
-        release_settings = await settings.list_release_matches(repos)
-    else:
-        branches = release_settings = logical_settings = None
-        default_branches = {}
+
+    settings = Settings.from_request(request, account, prefixer)
+    (branches, default_branches), logical_settings, release_settings = await gather(
+        BranchMiner.load_branches(
+            repos, prefixer, account, meta_ids, mdb, pdb, cache, strip=True,
+        ),
+        settings.list_logical_repositories(repos),
+        settings.list_release_matches(),
```

Comment:
@gaetano-guerriero I think that `settings.list_release_matches(repos)` will automatically include the logical repos. We can change the code, it seems